source/mergeLeft.js

```diff
@@ -0,0 +1,30 @@
+import _objectAssign from './internal/_objectAssign';
+import _curry2 from './internal/_curry2';
+
+
+/**
+ * Create a new object with the own properties of the first object merged with
+ * the own properties of the second object. If a key exists in both objects,
+ * the value from the first object will be used.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @sig {k: v} -> {k: v} -> {k: v}
+ * @param {Object} l
+ * @param {Object} r
+ * @return {Object}
+ * @see R.mergeRight, R.mergeDeepLeft, R.mergeWith, R.mergeWithKey
+ * @example
+ *
+ *      R.mergeLeft({ 'age': 40 }, { 'name': 'fred', 'age': 10 });
+ *      //=> { 'name': 'fred', 'age': 40 }
+ *
+ *      var resetToDefault = R.mergeLeft({x: 0});
+ *      resetToDefault({x: 5, y: 2}); //=> {x: 0, y: 2}
+ * @symb R.mergeLeft({ x: 1, y: 2 }, { y: 5, z: 3 }) = { x: 1, y: 2, z: 3 }
```

Comment:
hmm, I see some of the other `merge` functions have `@symb` definitions like this.. this doesn't really feel like it is in the spirit of `@symb` to me, can we just leave these off? Anyone else thoughts?

Reply:
Yeah I just copied that stuff from `merge`. In fact I don't even know what `@symb` does, it doesn't seem to show up in the docs… what is it for?

Meanwhile I'll go ahead and remove it 👍

Reply:
Looking at other `@symb` examples, it looks like it's more about giving generic (symbolic?) definitions, so maybe showing the ES6 equivalent or something would be useful? I.e.:

```js
@symb R.mergeLeft(a, b) = { ...b, ...a }
```

…or maybe that's just confusing. Anyway I've completely removed the `@symb` stuff for now.

Reply:
:+1: